package-lock.json

```diff
@@ -0,0 +1,6621 @@
+{
+    "name": "accessibility-snippets",
```

Comment:
can you remove this file :) we use yarn.lock

Reply:
also this should not be included: accessibility-snippets-0.0.15.vsix

would be awesome if you added these to the gitignore